### PR TITLE
6.0.20RNfixed broken links

### PR DESCRIPTION
**Pull request by AlonMagrafta**

### CODE REVIEW

**Review by lanceleonard — APPROVED** [review]

File: content/rs/release-notes/rs-6-0-20-april-2021.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ improvements:
     [Active-Active](https://docs.redislabs.com/latest/rs/administering/creating-databases/create-active-active/)
     Redis databases.
 
--   A new [migration process from regular Redis database to an Active-Active](https://docs.redislabs.com/latest/rs/administering/database-operations/migrate-to-active-active)
+-   A new [migration process from regular Redis database to an Active-Active](https://docs.redislabs.com/latest/rs/administering/database-operations/migrate-to-active-active/)
     Redis database using the Active-Passive (Replica Of) mechanism.
 
--   Support of [BITFIELD data type on Active-Active](https://docs.redislabs.com/latest/rs/references/developing-for-active-active/Strings-and-Bitfields-with-Active-Active-Databases)
+-   Support of [BITFIELD data type on Active-Active](https://docs.redislabs.com/latest/rs/references/developing-for-active-active/developing-strings-active-active/)
```
Comment (banker) [inline content/rs/release-notes/rs-6-0-20-april-2021.md:30]:
```suggestion
-   Support for the [BITFIELD data type on Active-Active](https://docs.redislabs.com/latest/rs/references/developing-for-active-active/developing-strings-active-active/)
```

File: content/rs/release-notes/rs-6-0-20-april-2021.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ improvements:
     [Active-Active](https://docs.redislabs.com/latest/rs/administering/creating-databases/create-active-active/)
     Redis databases.
 
--   A new [migration process from regular Redis database to an Active-Active](https://docs.redislabs.com/latest/rs/administering/database-operations/migrate-to-active-active)
+-   A new [migration process from regular Redis database to an Active-Active](https://docs.redislabs.com/latest/rs/administering/database-operations/migrate-to-active-active/)
```
Comment (banker) [inline content/rs/release-notes/rs-6-0-20-april-2021.md:27]:
```suggestion
-   A new [migration process for Active-Active](https://docs.redislabs.com/latest/rs/administering/database-operations/migrate-to-active-active/)
```

**Review by banker — COMMENTED** [review]
Couple of minor fixes here

**Review by kaitlynmichael — APPROVED** [review]
tested each new link

**Pull request merged**